### ISSUE TITLE
Empty declaration file when processing entry points using export  = class

### ISSUE THREAD

**Issue by endymion00** [timocov/dts-bundle-generator#42]
I'm developing a npm module and I wanted to generate an index.d.ts for including it in the module. 
My entry point is like this:

src/index.ts:
```
import { Cache } from "./lib/main";
export = Cache;
```
Cache is the main class and is placed in /lib/main/cache.ts
The export = Cache is the only way I found that allows to use the library in the same way in both Typescript and Javascript:
```
const Cache = require('mylib');
const cache = new Cache();
```

When executing 
`dts-bundle-generator -o dtsbundlegeneratortest1.d.ts src/index.ts` the output file is empty.
But if I execute:
`dts-bundle-generator -o dtsbundlegeneratortest1.d.ts src/cache.ts` the output file generates all typings (including public methods and properties but also the private ones...)

**Comment by timocov** [comment]
Hi,

> The export = Cache is the only way I found that allows to use the library in the same way in both Typescript and Javascript

But why not `export { Cache } from './lib/main';` (or even `export * from './lib/main';`) and then `import { Cache } from 'mylib';`?

But anyway, it seems that it is easy to reproduce, but I need some time to investigate this problem and figure out the solution.

**Comment by endymion00** [comment]
@timocov export = Cache is the only way I found (after researching a lot) that allows to use the library in in the same way in Javascript and Typescript.
If I use `export { Cache } from './lib/main'`  or `export * from './lib/main'` then in Typescript it works but when transpiled to Javascript I must make an instance in a different way:
```
const Cache = require('easy-cache').Cache;
const cache = new Cache();
```

or 
```
const Cache = require('easy-cache');
const cache = new Cache.Cache();
```

**Comment by timocov** [comment]
Good news: I found the reason of this bug - it is wrong detecting that some node is exported, if it is exported via `export =`.

Bad news: this bug can be fixed after (or together with) #17.
